docs/content/contribution-guidelines/operations/docs-styleguide.md

```diff
@@ -25,21 +25,24 @@ The goal of written documentation should be "just enough." Enough detail that th
 
 It should also be **clear**: terms should be defined and jargon explained when first used. Long, convoluted explanations should be examined to see if they might be better broken up into smaller chunks.
 
-Much of KubeStellar's existing documentation -- especially introductory and overview text -- is written with a slightly light and breezy tone. That is deliberate; it, however, does not mean that the texts should not be technically accurate.
+Much of KubeStellar's existing documentation -- especially introductory and overview text -- is written with a slightly light and breezy tone. That is deliberate it; however, does not mean that the texts should not be technically accurate.
 
 ### Avoid Use of Emojis in Prose
 
-The **Design Guide** will include guidance on use and placement of iconography in KubeStellar docs and UIs.
-In textual documentation, however, emojis may interfere with rendering and navigation of the website documentation, _especially_ if they are used in headings or titles. They also may make the documenation inaccessible to visitors who must use screen reader technology.
+The **Design Guide** will include guidance on the use and placement of iconography in KubeStellar docs and UIs.
+In textual documentation, however, emojis may interfere with rendering and navigation of the website documentation, _especially_ if they are used in headings or titles. They also may make the documentation inaccessible to visitors who must use screen reader technology.
 
 ### Always include alt-text for images
 
-This is a basic requirement for accessibility. Images and diagrams should always have an appropriate alt-text attribute. In general, keep the [W3C Guidelines for Accessibility](https://www.w3.org/TR/WCAG21/) in mind when writing/illustrating KubeStellar documentation 
+This is a basic requirement for accessibility. Images and diagrams should always have an appropriate alt-text attribute. In general, keep the [W3C Guidelines for Accessibility (WCAG 2.1)](https://www.w3.org/TR/WCAG21/) in mind when writing/illustrating KubeStellar documentation.
 
 ### Use Care If Using Generative AI
 
-The use of generative AI for writing assistance is becoming more common. Any text created via genAI should be **carefully** reviewed to make sure it is both coherent and accurate. It should also be checked to ensure it is not plagiarism, as some LLMs have been trained on copyrighted works.
+The use of generative AI for writing assistance is becoming more common. Any text created via generative AI should be **carefully** reviewed to ensure it is both coherent and accurate. It should also be checked to ensure it is not plagiarism, as some LLMs have been trained on copyrighted works.
 
-## Always Do a Spelling and Grammar Check
+## Always Perform a Spelling and Grammar Check

```

Comment:
In my opinion, "Perform" and "Do" are equally functional here. But "perform" is a longer word.